doc/dev/kdb-mount.md

```diff
@@ -0,0 +1,204 @@
+<!-- TODO: not an appropriate place for this type of docs -->
+
+# `kdb mount` Command
+
+## Basics: Direct Mode
+
+The `kdb mount` command is used to mount a new backend at some mountpoint.
+The basic usage of the command is:
+
+```
+kdb mount [OPTIONS...] <MOUNTPOINT> <BACKEND_PLUGIN> [...]
```

Comment:
I don't like the "anything might get passed to the backend plugin". We should either do something like `mount (2)` to have key=value options and/or have `kdb mount-backend` scripts which do something backend specific.

It must be properly documented for users which options are taken by which backend, similar to `mount (2)`.